engine/src/io/data_parser/CSVParser.cpp

```diff
@@ -92,7 +91,8 @@ std::unique_ptr<ral::frame::BlazingTable> csv_parser::parse(
 	}
 
 	if(file == nullptr) { 
-		return create_empty_table(schema.get_names(), schema.get_dtypes(), column_indices);
+		// return create_empty_table(schema.get_names(), schema.get_dtypes(), column_indices);  // do we need to create an empty table that has metadata?
```

Comment:
file should never really be nullptr. Find out if anyone knows why it exists

Reply:
Yeah, I dont know why either, we can easily take it out

Reply:
That was done over 5 month ago, i doubt if anyone will remember